e2e/test-resources/linodemachinetemplate.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha1
+kind: LinodeMachineTemplate
+metadata:
+    name: ($cluster)
+spec:
+    template:
+        spec:
+            region: us-sea
```

Comment:
we should probably pass in `(env('LINODE_REGION'))` for region here

Reply:
done